backend/app/services/implementations/log_records_service.py

```diff
@@ -47,19 +47,20 @@ def to_json_list(self, logs):
                         "attn_to": log[5],
                         "note": log[6],
                         "tags": log[7],
-                        "building": log[8],
-                        "employee_first_name": log[9],
-                        "employee_last_name": log[10],
-                        "attn_to_first_name": log[11],
-                        "attn_to_last_name": log[12],
+                        "building_id": log[8],
+                        "building": log[9],
+                        "employee_first_name": log[10],
+                        "employee_last_name": log[11],
+                        "attn_to_first_name": log[12],
+                        "attn_to_last_name": log[13],
                     }
                 )
             return logs_list
         except Exception as postgres_error:
             raise postgres_error
 
-    def filter_by_building(self, building):
-        return f"\nlogs.building='{building}'"
+    def filter_by_building_id(self, building_id):
```

Comment:
so because this should be an array you can change the code here too:
```
if type(employee_id) == list:
  sql_statement = f"\building_id={building_id[0]}"
  for i in range(1, len(building_id)):
    sql_statement = sql_statement + f"\nOR building_id={building_id[i]}"
  return sql_statement
return f"\nbuilding_id={building_id}"
```